core-bundle/src/Resources/contao/library/Contao/Config.php

```diff
@@ -404,16 +363,6 @@ public static function has($strKey)
 	 */
 	public static function get($strKey)
 	{
-		if ($newKey = self::getNewKey($strKey))
```

Comment:
Same here, I would leave the framework as we still have some `TL_CONFIG` keys.

Reply:
Restored in 7f5a64fa3ad7674bbc56c76911f7162433799619